pkg/store/store_test.go

```diff
@@ -296,185 +297,83 @@ func TestList(t *testing.T) {
+	createNodeToCell(s, auxNodeToCell{srcID: "2001", tgtID: "87893172902445057", labels: map[string]string{}})
+	createNodeToCell(s, auxNodeToCell{srcID: "2001", tgtID: "87893172902445058", labels: map[string]string{}})
+	createNodeToCell(s, auxNodeToCell{srcID: "2001", tgtID: "87893172902445059", labels: map[string]string{}})
+	createNodeToCell(s, auxNodeToCell{srcID: "2001", tgtID: "878931729024450560", labels: map[string]string{}})
+}
+
+func createNode(s Store, a auxNode) {
+	s.Create(context.TODO(), &topoapi.Object{
+		ID:     topo.ID(a.id),
 		Type:   topoapi.Object_ENTITY,
 		Obj:    &topoapi.Object_Entity{Entity: &topoapi.Entity{KindID: topoapi.ID("e2-node")}},
-		Labels: map[string]string{"env": "production"},
+		Labels: a.labels,
 	})
-	_ = s.Create(context.TODO(), &topoapi.Object{
-		ID:     "87893172902461441",
-		Type:   topoapi.Object_ENTITY,
-		Obj:    &topoapi.Object_Entity{Entity: &topoapi.Entity{KindID: topoapi.ID("e2-cell")}},
-		Labels: map[string]string{"env": "production"},
-	})
-	_ = s.Create(context.TODO(), &topoapi.Object{
-		ID:     "87893172902461443",
+}
+func createCell(s Store, a auxCell) {
+	s.Create(context.TODO(), &topoapi.Object{
+		ID:     topo.ID(a.id),
 		Type:   topoapi.Object_ENTITY,
 		Obj:    &topoapi.Object_Entity{Entity: &topoapi.Entity{KindID: topoapi.ID("e2-cell")}},
-		Labels: map[string]string{"env": "dev"},
+		Labels: a.labels,
 	})
-	_ = s.Create(context.TODO(), &topoapi.Object{
-		ID:     "87893172902445057",
+}
+func createNodeToCell(s Store, a auxNodeToCell) {
+	s.Create(context.TODO(), &topoapi.Object{
+		ID:     topo.ID(a.srcID + "-" + a.tgtID),
 		Type:   topoapi.Object_ENTITY,
-		Obj:    &topoapi.Object_Entity{Entity: &topoapi.Entity{KindID: topoapi.ID("e2-cell")}},
-		Labels: map[string]string{"env": "dev"},
+		Obj:    &topoapi.Object_Relation{Relation: &topoapi.Relation{KindID: topoapi.ID("e2-node-cell"), SrcEntityID: topoapi.ID(a.srcID), TgtEntityID: topoapi.ID(a.tgtID)}},
+		Labels: a.labels,
 	})
-	_ = s.Create(context.TODO(), &topoapi.Object{
-		ID:     "87893172902445058",
-		Type:   topoapi.Object_ENTITY,
-		Obj:    &topoapi.Object_Entity{Entity: &topoapi.Entity{KindID: topoapi.ID("e2-cell")}},
-		Labels: map[string]string{},
-	})
-	_ = s.Create(context.TODO(), &topoapi.Object{
-		ID:     "87893172902445059",
+}
+
+// creates both ways
+func createCellNeigbors(s Store, a auxCellNeigbor) {
+	s.Create(context.TODO(), &topoapi.Object{
+		ID:     topo.ID(a.srcID + "-" + a.tgtID),
 		Type:   topoapi.Object_ENTITY,
-		Obj:    &topoapi.Object_Entity{Entity: &topoapi.Entity{KindID: topoapi.ID("e2-cell")}},
-		Labels: map[string]string{},
+		Obj:    &topoapi.Object_Relation{Relation: &topoapi.Relation{KindID: topoapi.ID("e2-cell-neigbor"), SrcEntityID: topoapi.ID(a.srcID), TgtEntityID: topoapi.ID(a.tgtID)}},
+		Labels: a.labels,
 	})
-	_ = s.Create(context.TODO(), &topoapi.Object{
-		ID:     "87893172902445060",
+	s.Create(context.TODO(), &topoapi.Object{
```

Comment:
In golang you cannot skip the error without using _ . Assert the error  here.

Reply:
I've added the assert now.